artifactory/buildinfo/buildinfo.go

```diff
@@ -21,6 +21,7 @@ const (
 	Nuget   ModuleType = "nuget"
 	Go      ModuleType = "go"
 	Pip     ModuleType = "pip"
+	Pipenv  ModuleType = "pipenv"
```

Comment:
I think that the following two module types - 
```
	Pip     ModuleType = "pip"
	Pipenv  ModuleType = "pipenv"
```
should be merged into one module type - 
```
	Python  ModuleType = "python"
```
This is because the module type should correspond to the repository type. Similarly, both the "nuget" and "dotnet core" clients create the "nuget" module type, because both deploy artifacts to nuget repositories.